src/services/print.js

```diff
@@ -408,7 +410,9 @@ ngeo.Print.prototype.encodeVectorStyle_ =
   var imageStyle = style.getImage();
   var strokeStyle = style.getStroke();
   var textStyle = style.getText();
-  if (!goog.isNull(fillStyle)) {
+  var isLine = geometryType === 'LineString' ||
+          geometryType === 'MultiLineString';
```

Comment:
Please use `ol.geom.GeometryType.LINE_STRING` and `ol.geom.GeometryType.MULTI_LINE_STRING`. And add `goog.include('ol.geom.GeometryType')`.